File: Problemas/935A - Fafa And His Company/935A_Fafa_And_His_Company.py
# 935A - Fafa and his Company 
# https://codeforces.com/problemset/problem/935/A

# Input:
# 1) empleados = Número de empleados 
empleados = int(input())

# Contador de combinaciones de repartición TL-Empleado
reparticion = 0

# Itera sobre las posibles cantidades de Team Leader
for TL in range(1, empleados // 2 + 1):

    # Si se pueden repartir equitativamente los empleados entre los Team Leaders, entonces suma +1 al contador de reparticiones
    if (empleados - TL) % TL == 0:
        reparticion += 1

# Imprime la cantidad de combinaciones de repartición TL-Empleado
print(reparticion)

File: Problemas/935A - Fafa And His Company/935A_Fafa_And_His_Company.cpp
// 935A - Fafa And His Company  
// https://codeforces.com/problemset/problem/935/A

#include <stdio.h>
#include <stdlib.h>
#include <bits/stdc++.h>
#include <cmath>
using namespace std;

int main(){

    // Input:
    // 1) empleados = Número de empleados en la empresa de Fafa
    int empleados;
    cin >> empleados;

    // Contador de combinaciones de repartición TL-Empleado
    int reparticiones = 0;

    // Itera sobre las posibles cantidades de Team Leader
    for(int TL = 1; TL < (empleados / 2 + 1); TL++) {

        // Si se pueden repartir equitativamente los empleados entre los Team Leaders, entonces suma +1 al contador de reparticiones
        if((empleados - TL) % TL == 0) {
            reparticiones++;
        }
    }

    // Imprime la cantidad de combinaciones de repartición TL-Empleado
    cout << reparticiones << endl;


    return 0;
}